src/rf4ce/ctrlm_rf4ce_network.h

```diff
@@ -67,6 +67,9 @@
 #define IR_RF_DATABASE_STATUS_DB_DOWNLOAD_YES           (0x80)
 #define IR_RF_DATABASE_STATUS_RESERVED                  (0x20)
 
+#define RF4CE_RCU_ID_RANGE_MIN (NETWORK_ID_BASE_RF4CE)
+#define RF4CE_RCU_ID_RANGE_MAX (RF4CE_RCU_ID_RANGE_MIN + CTRLM_MAIN_MAX_BOUND_CONTROLLERS)
```

Comment:
```suggestion
#define RF4CE_RCU_ID_RANGE_MAX (RF4CE_RCU_ID_RANGE_MIN + CTRLM_MAIN_MAX_BOUND_CONTROLLERS-1)
```
Should this be the same as BLE where its inclusive and MAX ID is a valid ID?